client-js/app/scripts/manager.ts

```diff
@@ -53,7 +61,7 @@ export interface ManagerApi {
   serverDisconnected: Observable<ServerDisconnectMessage>;
 }
 
-export class ManagerServiceJsonRpc extends rpc.RpcService implements ManagerApi {
+export class ManagerApiImpl extends rpc.RpcService implements ManagerApi {
```

Comment:
It might be better to avoid fragmenting standards already in place until we are ready to actually replace them all. 

If the timeline for a PR fixing all of something is more than 1-2 days away, I think it should follow the existing naming convention then replace it when the new PR is finally ready.